modules/Calendar/Calendar.php

```diff
@@ -46,10 +46,18 @@
 require_once('modules/Calendar/CalendarUtils.php');
 require_once('modules/Calendar/CalendarActivity.php');
 
-
 class Calendar {
-	
-	public $view = 'week'; // current view
+
+	public $activityList = array("FP_events" => array("showCompleted" => true,"start" =>  "date_start", "end" => "date_end"),
+								 "Meeting" => array("showCompleted" => true,"start" =>  "date_start", "end" => "date_end"),
+								 "Call" => array("showCompleted" => true,"start" =>  "date_start", "end" => "date_end"),
+								 "Task" => array("showCompleted" => true,"start" =>  "date_start", "end" => "date_due"),
+//								 "ProjectTask" => array("showCompleted" => true,"start" =>  "date_start", "end" => "date_finish"),
```

Comment:
Again - commented out code - yes we may use it later but that doesn't mean we should clog up the system now.